app/assets/javascripts/miq_flash.js

```diff
@@ -35,6 +39,17 @@ function add_flash(msg, level, options) {
   var alertDiv = $('<div class="' + cls.alert + '"><button class="close" data-dismiss="alert"><span class="pficon pficon-close"></span></button></div>');
   alertDiv.append(iconSpan, textStrong);
 
+  // if options.long alert is given than add a `See More` button to the alert div to allow user the get more details of the error/alert.
+  if (options.long_alert) {
+    var detailsButtonTxt = "See More";
```

Comment:
please extract text....
also it might not be in a bottom (/cc @serenamarie125 )

Reply:
*button